packages/toasts/src/ToastsProvider.test.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useContext } from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import {
+  render,
+  screen,
+  fireEvent,
+} from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import ToastsProvider, { ToastsContext } from './ToastsProvider';
+import Toasts from './Toasts';
+import enTranslations from './en.json';
+
+const MockButtons = () => {
+  const { addToast, removeToast } = useContext(ToastsContext);
+  return (
+    <>
+      <Toasts />
+      <button type="button" onClick={() => addToast('Butter')}>Add</button>
+      <button type="button" onClick={() => removeToast(0)}>Remove</button>
+    </>
+  );
+};
```

Comment:
Let's consolidate these two components into one wrapper component. maybe something like
```
const Wrapper = () => 
  <IntlProvider>
    <ToastsProvider>
        <Toasts />
        <button type="button" onClick={() => {
          addToast('Butter');
          addToast('Github', { label: 'Test', href: 'https://github.com/' })
        }}>Add</button>
        <button type="button" onClick={() => removeToast(0)}>Remove</button>
    </ToastsProvider>
  </IntlProvider>
```

Reply:
For this one, the test kept throwing an error regarding React.children.only was expecting a single child. Further looking into this error concluded it had to do with testing a button